presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -571,13 +589,15 @@ Procedures
     * ``DROP``: drop any partitions that exist in the metastore but not on the file system.
     * ``FULL``: perform both ``ADD`` and ``DROP``.
 
-Examples
```

Comment:
Keep the section header as "Examples", since these are all the examples of using the Hive connector, not just about accessing external data.